app/views/docs/database.phtml

```diff
@@ -453,5 +453,183 @@ func main() async throws {
     </tbody>
 </table>
 
+<h3><a href="/docs/database#pagination" id="ordering">Ordering Results</a></h3>
+<p>When querying using the <a href="/docs/server/database#databaseListDocuments">listDocuments</a> endpoint, you can specify the order of the documents returned by providing the <b>orderAttributes</b> and <b>orderTypes</b> parameters. The results will be sorted by <b>$id</b> (create date) in ascending order when no order parameters are provided.</p>
+
+<p>The parameter <b>orderAttributes</b> takes a string array of attributes IDs. The <b>orderTypes</b> parameter takes a string array equal in length to <b>orderAttributes</b> that indicate if each attribute should be sorted in ascending or descending order. Ascending and descending is denoted as <span class="tag">"ASC"</span> and <span class="tag">"DESC"</span> respectively.</p>
+
+
+<ul class="phases clear" data-ui-phases>
+    <li>
+        <h3>Web</h3>
+        <div class="ide" data-lang="javascript" data-lang-label="Web SDK">
+            <pre class="line-numbers"><code class="prism language-javascript" data-prism>let promise = appwrite.database.listDocuments('movies', [], 25, 0, '', 'after', ['title'], ['ASC']);</code></pre>
```

Comment:
Let's split web code into multiple lines so people don't need to scroll. I think.. Because in 1-liner you wont see the end of the command, the badge `web` will cover it.

Reply:
That sounds like and UI problem 💢 
I'll split this into multiple lines :)